recipes/libsmu/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "libsmu" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/analogdevicesinc/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 290d72bf5cec9ceab0030f6a4bc49bfd962aff56997aebebf70a20cb4a6bf77a
+
+build:
+  number: 1
+  run_exports:
+    - {{ pin_subpackage('libsmu', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - {{ cdt('libudev-devel') }}  # [linux]
+    - make  # [not win]
+    - ninja  # [win]
+    - pkg-config  # [not win]
+  host:
+    - boost-cpp
+    - libusb >=1
+    - python
+    - setuptools
+    - cython
+  run:
+    - python
+
+test:
+  commands:
+    # verify that executables can be run
+    - smu -h
+
+    # verify that headers get installed
+    - test -f $PREFIX/include/libsmu/libsmu.hpp  # [not win]
+    - if not exist %PREFIX%\\Library\\include\\libsmu\\libsmu.hpp exit 1  # [win]
+
+    # verify that libraries get installed
+    - test -f $PREFIX/lib/libsmu${SHLIB_EXT}  # [not win]
+    - if not exist %PREFIX%\\Library\\bin\\libsmu.dll exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\libsmu.lib exit 1  # [win]
+  imports:
+    - pysmu
+
+about:
+  home: https://github.com/analogdevicesinc/libsmu
+  license: BSD
```

is that bsd 3? if so, please change it to
```suggestion
  license: BSD-3-Clause
```